test/parallel/test-http2-binding.js

```diff
@@ -13,7 +13,7 @@ assert(http2.SessionShutdownWrap);
 assert.strictEqual(typeof http2.Http2Session, 'function');
 assert.strictEqual(typeof http2.SessionShutdownWrap, 'function');
 
-const settings = http2.getDefaultSettings({});
+const settings = require('http2').getDefaultSettings({});
```

Comment:
This test was failing otherwise. Now matches the code just below: https://github.com/sebdeckers/http2/blob/430f9a648163b612a2240151b41e3639c4a44958/test/parallel/test-http2-binding.js#L28